starburst-presto/presto.sh

```diff
@@ -123,23 +122,47 @@ function configure_hive(){
 connector.name=hive-hadoop2
 hive.metastore.uri=${metastore_uri}
 EOF
+
+# Add new connectors configs here 
+# For example: Adding jmx properties like
+  cat > presto-server/etc/catalog/tpch.properties <<- "EOF"
+connector.name=tpch
+tpch.splits-per-node=4
+EOF
+
+cat > presto-server/etc/catalog/tpcds.properties <<- "EOF"
+connector.name=tpcds
+EOF
+
+cat > presto-server/etc/catalog/jmx.properties <<- "EOF"
```

Comment:
nit: please use `<<` rather then `<<-`. The addition of `-` causes stripping of leading whitespace, which is not wanted here.

Reply:
Okay!